src/commons/modifiers.ts

```diff
@@ -97,8 +97,8 @@ export type ContainerModifiers =
   BackgroundColorModifier;
 
 export function extractColorValue(props: Dictionary<any>) {
-  const scheme = Appearance.getColorScheme() || 'light';
-  const schemeColors = Colors.schemes[scheme];
+  const scheme = Colors.currentScheme === 'default' ? Appearance.getColorScheme() : Colors.currentScheme;
+  const schemeColors = Colors.schemes[scheme || 'light'];
```

Comment:
I think it's better to add a `getScheme` method in `Colors` just to to avoid repeating this piece of code